packages/yoroi-extension/app/ergo-connector/components/signin/AddCollateralPage.js

```diff
@@ -0,0 +1,295 @@
+/* eslint-disable no-nested-ternary */
+// @flow
+import { Component } from 'react';
+import type { Node } from 'react';
+import { intlShape, defineMessages, FormattedHTMLMessage } from 'react-intl';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import { Button, Typography, Alert } from '@mui/material';
+import TextField from '../../../components/common/TextField';
+import globalMessages from '../../../i18n/global-messages';
+import { observer } from 'mobx-react';
+import config from '../../../config';
+import vjf from 'mobx-react-form/lib/validators/VJF';
+import ReactToolboxMobxForm from '../../../utils/ReactToolboxMobxForm';
+import { splitAmount, truncateToken } from '../../../utils/formatters';
+import type { TokenLookupKey, TokenEntry } from '../../../api/common/lib/MultiToken';
+import type { TokenRow } from '../../../api/ada/lib/storage/database/primitives/tables';
+import {
+  getTokenName,
+  getTokenIdentifierIfExists,
+  assetNameFromIdentifier,
+} from '../../../stores/stateless/tokenHelpers';
+import BigNumber from 'bignumber.js';
+import ExplorableHashContainer from '../../../containers/widgets/ExplorableHashContainer';
+import { SelectedExplorer } from '../../../domain/SelectedExplorer';
+import type { CardanoConnectorSignRequest, SignSubmissionErrorType } from '../../types';
+import { Box } from '@mui/system';
+import { signTxMessages } from './SignTxPage';
+import { WrongPassphraseError } from '../../../api/ada/lib/cardanoCrypto/cryptoErrors';
+import { LoadingButton } from '@mui/lab';
+import { ReactComponent as AddCollateralIcon } from '../../../assets/images/dapp-connector/add-collateral.inline.svg';
+
+type Props = {|
+  +txData: ?CardanoConnectorSignRequest,
+  +onCancel: () => void,
+  +onConfirm: string => Promise<void>,
+  +getTokenInfo: ($ReadOnly<Inexact<TokenLookupKey>>) => ?$ReadOnly<TokenRow>,
+  +selectedExplorer: SelectedExplorer,
+  +submissionError: ?SignSubmissionErrorType,
+|};
+
+const messages = defineMessages({
+  incorrectWalletPasswordError: {
+    id: 'api.errors.IncorrectPasswordError',
+    defaultMessage: '!!!Incorrect wallet password.',
+  },
+  reorgTitle: {
+    id: 'connector.signin.reorg.title',
+    defaultMessage: '!!!Add Collateral',
+  },
+  reorgMessage: {
+    id: 'connector.signin.reorg.message',
+    defaultMessage:
+      '!!!<span>To interact with <a href="https://emurgohelpdesk.zendesk.com/hc/en-us/articles/4415793858959-What-are-smart-contracts" target="_blank">smart contract</a> in Cardano you should add collateral, which means to make a 0 ADA transaction.<br/><br/>It is a guarantee that prevent from failing smart contracts and scams. <a href="https://docs.cardano.org/plutus/collateral-mechanism" target="_blank">Learn more</a> about collateral.</span>',
```

Comment:
Notice that the `a` tag and the `href` will not be translated! take a look [here](https://github.com/Emurgo/yoroi-frontend/blob/bcc07d8536e85a4c61a70f9282538d69a479f1a7/packages/yoroi-extension/app/components/loading/Crashed.js#L63,L79) and see how you can remove them and at the same time integrate it with the reset of the text

Reply:
Great! Thank you, pushing an update.